app/assets/stylesheets/ama_layout/layout_components/button-grouping.scss

```diff
@@ -61,6 +61,7 @@
 
   &__content{
     @include absolute-center;
+    -webkit-transform: translateX(-50%) translateY(-50%);
```

Comment:
The `absolute-center` mixin comes from Foundation.

There's actually an issue with the code in Foundation core: https://github.com/zurb/foundation-sites/pull/9077